Casks/p/psi-plus.rb

```diff
@@ -9,8 +9,8 @@
   homepage "https://psi-plus.com/"
 
   livecheck do
-    url "https://sourceforge.net/projects/psiplus/files/macOS/tehnick/"
-    regex(/Psi%2B[._-]?(\d+(?:\.\d+)+)[._-]?macOS.*-x86_64\.dmg/i)
+    url "https://sourceforge.net/projects/psiplus/rss?path=/macOS/tehnick"
+    regex(%r{url=.*?/macOS/tehnick/Psi%2B[._-]?v?(\d+(?:\.\d+)+)[._-]?macOS[^"' >]*?\.dmg}i)
```

Comment:
```suggestion
    regex(%r{url=.*?/Psi%2B[._-]?v?(\d+(?:\.\d+)+)[._-]?macOS[^"' >]*?\.dmg}i)
```

We're already restricting the RSS feed to the `/macOS/tehnick` directory in the `url` (i.e., `path=/macOS/tehnick`), so we can omit it from the regex and `.*?` will cover it.

Reply:
The path was added to keep the same functionality, as we did not previously match entries in `/macOS/tehnick/Archive`. In retrospect, there should be no need to filter these. Thanks!